### DISCUSSION TITLE
Possible to backup settings?

### DISCUSSION THREAD

**Discussion by bossanovaorca** [birchill/10ten-ja-reader#1429] [Q&A]
Hello! Is it possible to backup my extension settings to a config file or to cloud storage for  easier setup on different devices?

**Comment by birtles** [comment]
Hi @bossanovaorca!

I'm afraid there's not currently a function to export/import a config file. Which browser are you using?

For most browsers like Firefox and Chrome, if you sign in to your browser your settings should be synchronized. For example, on Firefox, if you create a Firefox/Mozilla account and sign in to Firefox, when you sign in on your new computer your extension settings will be synchronized automatically.

There are only a few settings that we choose not to synchronize such as whether or not mouse interactivity is enabled and whether or not the puck is shown since some people may want to set these differently on different devices. Also, I'm not sure if the shortcut key for activating 10ten is synchronized since that is controlled by the browser (I think it is synchronized, however).

Does that help?
